packages/tools/devtools/devtools-core/package.json

```diff
@@ -40,8 +82,10 @@
 		"rebuild": "npm run clean && npm run build",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha \"dist/**/*.test.js\" --exit",
-		"tsc": "tsc",
+		"test:mocha": "npm run test:mocha:esm && npm run test:mocha:cjs",
+		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"tsc": "fluid-tsc commonjs --project ./tsconfig.cjs.json && copyfiles -f ../../../../common/build/build-common/src/cjs/package.json ./dist",
```

Comment:
Nit: it would probably be nice to capture `copyfiles -f ../../../../common/build/build-common/src/cjs/package.json ./dist"` as a separate script, with a name that captures why it's needed. Without additional context, it's very confusing why we're copying a `package.json` from another package into one of our build directories.